cli/tests/install.bats

```diff
@@ -367,34 +367,48 @@ EOF
   )"
 }
 
-@test "resolution message: single package not availabe on all systems" {
+# bats test_tags=install:single-not-on-all-systems
+@test "resolution fixub: package not available on all systems installs with looser constraints" {
   "$FLOX_BIN" init
 
   _FLOX_USE_CATALOG_MOCK="$GENERATED_DATA/resolve/bpftrace.json" \
     run "$FLOX_BIN" install bpftrace
 
-  assert_failure
-  assert_output "$(
-    cat << EOF
-❌ ERROR: resolution failed: package 'bpftrace' not available for
-    - aarch64-darwin
-    - x86_64-darwin
-  but it is available for
-    - aarch64-linux
-    - x86_64-linux
-
-For more on managing system-specific packages, visit the documentation:
-https://flox.dev/docs/tutorials/multi-arch-environments/#handling-unsupported-packages
-EOF
-  )"
+  assert_success
```

Comment:
nit: is it worth making any messaging assertions?

Reply:
do you mean

```suggestion
  assert_success
  assert_output --partial "✅ 'bpftrace' installed to environment"
```